### PR TITLE
[Ubuntu] Rework Git installation to avoid using apt repo

### PR DESCRIPTION
**Pull request by dibir-magomedsaygitov**
# Description
In scope of this PR we are reworking `git` and `git-lfs` to avoid using apt repo.

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/git.sh [hunk 1]
```diff
@@ -8,14 +8,21 @@
...
+latest_git_version=$(curl -s -L "https://api.github.com/repos/git/git/tags" | jq -r '.[].name' | grep -ve "-.*" | head -1 | tr -d "v")
+download_with_retries https://github.com/git/git/archive/v${latest_git_version}.tar.gz "/tmp" "git.tar.gz"
+git_installation_path="/usr/local/git"
+git_tar_tmp="/tmp/git.tar.gz"
+mkdir -p "${git_installation_path}"
+tar -zxf "${git_tar_tmp}" -C "${git_installation_path}"
+rm -f "${git_tar_tmp}"
+cd "${git_installation_path}/git-${latest_git_version}"
+make prefix=/usr/local all
+make prefix=/usr/local install
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/git.sh:20]: Do we really need to build git from source code during image-generation? I don't think that it is good solution.
I think both `git` and `git-lfs` should be available in default apt sources without adding additional repos and installation will be faster

Comment (dibir-magomedsaygitov) [reply images/linux/scripts/installers/git.sh]: @maxim-lobanov thank you, simplified installation

File: images/linux/scripts/installers/git.sh [hunk 2]
```diff
@@ -8,14 +8,13 @@
...
 ## Install git
-add-apt-repository ppa:git-core/ppa -y
 apt-get update
 apt-get install git -y
 git --version
 
 # Install git-lfs
-curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-apt-get install -y --no-install-recommends git-lfs
+apt-get update -y
```
Comment (maxim-lobanov) [inline images/linux/scripts/installers/git.sh:16]: I think we don't need to run apt-get update one more time. We have already run it on line 11

**Review by maxim-lobanov — APPROVED** [review]

File: images/linux/scripts/installers/git.sh [hunk 3]
```diff
@@ -8,14 +8,12 @@
 source $HELPER_SCRIPTS/install.sh
 
 ## Install git
-add-apt-repository ppa:git-core/ppa -y
 apt-get update
 apt-get install git -y
```
Comment (miketimofeev) [inline images/linux/scripts/installers/git.sh:12]: is the version in the repo fresh enough? I think it's pretty outdated at least for ubuntu 16 & 18

File: images/linux/scripts/installers/git.sh [hunk 4]
```diff
@@ -8,14 +8,12 @@
...
 git --version
 
 # Install git-lfs
-curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
-apt-get install -y --no-install-recommends git-lfs
+apt-get install -y git-lfs
```
Comment (miketimofeev) [inline images/linux/scripts/installers/git.sh:16]: same here

**Review by al-cheb — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**